libp2p/__init__.py

```diff
@@ -105,6 +105,10 @@ def initialize_default_swarm(
     )
 
     peerstore = peerstore_opt or PeerStore()
+    # Store our key pair in peerstore
+    peerstore.add_pubkey(id_opt, key_pair.public_key)
+    peerstore.add_privkey(id_opt, key_pair.private_key)
```

Comment:
not blocking but if you like this here you go:

consider adding an `add_key_pair` method that takes a `key_pair` and adds both the pubkey and privkey internally? generally easier to get one call correct, rather than have to remember to call two methods when you need to do this.